tech.json

```diff
@@ -70,7 +70,7 @@
                 {
                   "name": "canSpaceJumpWaterEscape",
                   "requires": [ "canSpaceJumpWaterBounce" ],
-                  "note": "Converting Space Jump Water Bouncing into regular space jumps without Gravity. Typically requires HiJump."
+                  "note": "Converting Space Jump Water Bouncing into regular space jumps without Gravity. Requires HiJump unless the water level rises and falls."
```

Comment:
This tech should not include escaping without hijump while in tidal water. If it is possible, its much much harder and would need it's own tech.
```suggestion
                  "note": "Using HiJump to escape the waterline during a Space Jump Water Bounce.",
                  "devNote": "Escaping tidal water without HiJump could be added as a more advanced tech."
```